playbooks/utils/replace_vm_host.yml

```diff
@@ -25,26 +28,18 @@
         # show_cluster: true
         vm_name: "{{ replacement_vm }}"
       register: old_vm_info
-      tags: 
```

Comment:
@acozine I would prefer to leave a way to get the information about a host in the playbook.  Removing it means if I want to double check if the machine looks the same before and after the replacement I have to have access to vsphere.  Maybe the tag could be something like sys_info instead of disk

Reply:
My concern about tags is really about the Tower way of managing them. There's no good way to see which tags are available from the Tower side. You can add them to a template, but then the user has to delete them to get the full playbook to run. But maybe we can meet this use case with multiple Templates for the same playbook. I will investigate.

Reply:
We will add more diagnostics. Use case: we need to confirm that the new machine has the same CPU, memory, and amount of storage space as the old machine, for confidence that the replacement has been done correctly.

Reply:
Can we also hook this up to the pulsys user playbook?